website/src/pages/playground/index.tsx

```diff
@@ -0,0 +1,126 @@
+
+type TrailingComma = "none" | "all";
+
+const codeSample = `public interface MyInterface {
+  String foo();
+  int[] bar();
+}
+
+public abstract class Foo implements MyInterface {
+  @Override public String foo() {
+            // TODO: return an actual value here
+        return "hello";
+      }
+  @Override public int[] bar() {
+    return new int[] {  1,
+
+      2, 3,
+    };
+  }
+
+  public final static boolean baz(final String foo, final int bar, final boolean baz) {
+    return baz;
+  }
+}`;
+
+export default function Playground() {
+  const [printWidth, setPrintWidth] = useState(80);
+  const [tabWidth, setTabWidth] = useState(2);
+  const [useTabs, setUseTabs] = useState(false);
+  const [trailingComma, setTrailingComma] = useState<TrailingComma>("all");
+  const [requirePragma, setRequirePragma] = useState(false);
+  const [code, setCode] = useState(codeSample);
+  const [formattedCode, setFormattedCode] = useState("");
+
+  useMemo(() => {
+    prettier
+      .format(code, {
+        parser: "java",
+        plugins: [prettierPluginJava],
+        printWidth,
+        tabWidth,
+        useTabs,
+        trailingComma,
+        requirePragma
+      })
+      .then(setFormattedCode)
+      .catch(error => setFormattedCode(error.message));
+  }, [printWidth, tabWidth, useTabs, trailingComma, requirePragma, code]);
+
+  return (
+    <Layout noFooter title="Playground">
+      <main>
+        <div className={styles.playground}>
+          <div className={styles.options}>
+            <details open>
+              <summary>Global</summary>
+              <label title="The line length that the printer will wrap on.">
+                --print-width{" "}
+                <input
+                  type="number"
+                  min={0}
+                  value={printWidth}
+                  onChange={event => setPrintWidth(event.target.valueAsNumber)}
+                />
+              </label>
+              <label title="The number of spaces per indentation-level.">
+                --tab-width{" "}
+                <input
+                  type="number"
+                  min={0}
+                  value={tabWidth}
+                  onChange={event => setTabWidth(event.target.valueAsNumber)}
+                />
+              </label>
+              <label title="Indent lines with tabs instead of spaces.">
+                <input
+                  type="checkbox"
+                  checked={useTabs}
+                  onChange={event => setUseTabs(event.target.checked)}
+                />{" "}
+                --use-tabs
+              </label>
+            </details>
+            <details open>
+              <summary>Java</summary>
+              <label title="Print trailing commas wherever possible when multi-line.">
+                --trailing-comma{" "}
+                <select
+                  disabled={true} // this option isn't seen by prettier-plugin-java when passed programmatically
```

Comment:
It's strange that the other options are working but not this one, I wonder what causes this issue 🤔

Reply:
Figured this out -- just had to add `trailingComma` as an option in [`options.js`](https://github.com/jhipster/prettier-java/pull/628/files#diff-e239c88e07ba045568350e8839476ffe09f9f6d2c986a60e83b04aa188ea3203).

Reply:
Great !